mteb/tasks/Retrieval/kor/AutoRAGRetrieval.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class AutoRAGRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="AutoRAGRetrieval",
+        description="This dataset enables the evaluation of Korean RAG performance across various domains—finance, public sector, healthcare, legal, and commerce—by providing publicly accessible documents, questions, and answers.",
+        reference="https://arxiv.org/abs/2410.20878",
+        dataset={
+            "path": "yjoonjang/markers_bm",
+            "revision": "fd7df84ac089bbec763b1c6bb1b56e985df5cc5c",
+        },
+        type="Retrieval",
+        prompt="Retrieve text based on user query.",
+        category="s2p",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["kor-Hang"],
+        main_score="ndcg_at_10",
+        date=("2024-08-03", "2024-08-03"),
+        domains=["Government", "Medical", "Legal", "Social"],
+        task_subtypes=["Article retrieval"],
+        license="mit",
+        annotations_creators="human-annotated",
+        dialect=[],
+        sample_creation="created",
+        bibtex_citation="""@misc{kim2024autoragautomatedframeworkoptimization,
+      title={AutoRAG: Automated Framework for optimization of Retrieval Augmented Generation Pipeline}, 
+      author={Dongkyu Kim and Byoungwook Kim and Donggeon Han and Matouš Eibich},
+      year={2024},
+      eprint={2410.20878},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL},
+      url={https://arxiv.org/abs/2410.20878}, 
+}""",
+        descriptive_stats={
```

Comment:
You'll need to recalculate this since the current MTEB version doesn't have `descriptive_stats` as a field in TaskMetadata

Reply:
How should I add the data statistics?
Also, should I just run the test models by python code following:
```python
from mteb import MTEB
from sentence_transformers import SentenceTransformer

# Define the sentence-transformers model name
model_name = "sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2"

model = SentenceTransformer(model_name)
evaluation = MTEB(tasks=[AutoRAGRetrieval()])
```

Reply:
Same way as previous `calculate_metadata_metrics` and json file will be generated. Yes, you can run code like this, but I'll try to fix your issues with loading this task correctly